src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -1,12 +1,21 @@
 package com.minecolonies.coremod.event;
 
-import com.minecolonies.coremod.colony.Structures;
+import com.minecolonies.api.colony.permissions.Action;
+import com.minecolonies.api.util.BlockPosUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.BlockPosUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)